packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -523,17 +494,13 @@ contract PerpetualPositionManager is FundingRatePayer, FundingRateApplier {
      *        GLOBAL STATE FUNCTIONS        *
      ****************************************/
 
-    function applyFundingRate() external notEmergencyShutdown() updateFundingRate() nonReentrant() {
-        return;
-    }
-
     /**
      * @notice Premature contract settlement under emergency circumstances.
      * @dev Only the governor can call this function as they are permissioned within the `FinancialContractAdmin`.
      * Upon emergency shutdown, the contract settlement time is set to the shutdown time. This enables withdrawal
      * to occur via the `settleEmergencyShutdown` function.
      */
-    function emergencyShutdown() external override notEmergencyShutdown() updateFundingRate() nonReentrant() {
+    function emergencyShutdown() external override notEmergencyShutdown() fees() nonReentrant() {
```

Comment:
Wait, `emergencyShutdown` was not paying regular fees before, but now it is?

Reply:
Yeah, I think it should be, right?